src/catalogue/category/catalogueCategoryDialog.component.tsx

```diff
@@ -344,82 +329,91 @@ const CatalogueCategoryDialog = React.memo(
-                      catalogue_item_properties: formFields,
-                    })
-                  }
-                  nameFields={nameFields}
-                  onChangeNameFields={setNameFields}
-                  typeFields={typeFields}
-                  onChangeTypeFields={setTypeFields}
-                  errorFields={errorFields}
-                  propertyNameError={duplicatePropertyError}
-                  onChangePropertyNameError={setDuplicatePropertyError}
-                  onChangeErrorFields={setErrorFields}
-                  resetFormError={() => setFormError(undefined)}
-                />
-              </Box>
-            </Box>
-          )}
+                      is_leaf: value === 'true' ? true : false,
+                    };
+                    if (value === 'false') {
+                      newData.catalogue_item_properties = undefined;
+                      setErrorFields([]);
+                      setNameFields([]);
+                      setTypeFields([]);
+                    }
+                    handleFormChange(newData);
+                  }}
+                >
+                  <FormControlLabel
+                    value="false"
+                    control={<Radio />}
+                    label="Catalogue Categories"
+                  />
+                  <FormControlLabel
+                    value="true"
+                    control={<Radio />}
+                    label="Catalogue Items"
+                  />
+                </RadioGroup>
+              </FormControl>
+            </Grid>
+            {categoryData.is_leaf === true && (
+              <Grid item sx={{ alignItems: 'center', width: '100%' }}>
+                <Grid>
+                  <Divider sx={{ minWidth: '700px' }} />
+                </Grid>
+                <Grid item sx={{ paddingLeft: '8px', paddingTop: '24px' }}>
+                  <Typography variant="h6">Catalogue Item Fields</Typography>
+                  <CataloguePropertiesForm
+                    formFields={categoryData.catalogue_item_properties ?? []}
+                    onChangeFormFields={(
+                      formFields: CatalogueCategoryFormData[]
+                    ) =>
+                      handleFormChange({
+                        ...categoryData,
+                        catalogue_item_properties: formFields,
+                      })
+                    }
+                    nameFields={nameFields}
+                    onChangeNameFields={setNameFields}
+                    typeFields={typeFields}
+                    onChangeTypeFields={setTypeFields}
+                    errorFields={errorFields}
+                    propertyNameError={duplicatePropertyError}
+                    onChangePropertyNameError={setDuplicatePropertyError}
+                    onChangeErrorFields={setErrorFields}
+                    resetFormError={() => setFormError(undefined)}
+                  />
+                </Grid>
+              </Grid>
```

Comment:
```suggestion
              <>
                <Grid item>
                  <Divider sx={{ minWidth: '700px' }} />
                </Grid>
                <Grid item sx={{ paddingLeft: 1, paddingTop: 3 }}>
                  <Typography variant="h6">Catalogue Item Fields</Typography>
                  <CataloguePropertiesForm
                    formFields={categoryData.catalogue_item_properties ?? []}
                    onChangeFormFields={(
                      formFields: CatalogueCategoryFormData[]
                    ) =>
                      handleFormChange({
                        ...categoryData,
                        catalogue_item_properties: formFields,
                      })
                    }
                    nameFields={nameFields}
                    onChangeNameFields={setNameFields}
                    typeFields={typeFields}
                    onChangeTypeFields={setTypeFields}
                    errorFields={errorFields}
                    propertyNameError={duplicatePropertyError}
                    onChangePropertyNameError={setDuplicatePropertyError}
                    onChangeErrorFields={setErrorFields}
                    resetFormError={() => setFormError(undefined)}
                  />
                </Grid>
              </>
```
I don't think the outer grid is really necessary as it should be taken care of by the grid container above. There was also a missing `item`.